runtime/doc/options.txt

```diff
@@ -4364,6 +4364,50 @@ A jump table for the options with a short description can be found at |Q_op|.
 	    |nobody         |  3 nobody     |  0 nobody     |3   nobody
 	    |there          |  4 there      |  1 there      |  1 there
 
+				*'numbercolumn'* *'nuc'*
+'numbercolumn' 'nuc'	string	(default: empty)
+			local to window
+	EXPERIMENTAL
+	When this option is not empty, it determines the content of the number
+	column. The format of this option is like that of 'statusline'. The
+	variables *v:redraw_lnum* and *v:redraw_relnum* will be set to the to be
+	printed line number and relative line number respectively. These can be
+	evaluated in an expression inside a 'statusline' %{} block.
+
+	Some of the items from the 'statusline' format are different
+	'numbercolumn':
+
+	%l	v:redraw_lnum, can be used inside %{} block
+	%r	v:redraw_relnum, can be used inside %{} block
+	%s	Sign column for v:redraw_lnum
+	%C	Fold column for v:redraw_lnum
```

Comment:
The sign/fold column should be separated out of `v:redraw_lnum` which should just be an integer.

Reply:
It is. What I'm saying here is that `%s` and `%C` get the sign/fold column content for the currently drawn line, which is equal to `v:redraw_lnum`. Perhaps not too clear.

Reply:
Ok, then this just needs some minor rewording.